doc/api/tools_variable_substitutions.md

```diff
@@ -476,6 +476,13 @@ returns the current course's term start date.
 
 **Availability**: *when launched in a course that has a term with a start date*  
 
+```
+active
+```
+## Canvas.term.name
+returns the current course's term name.
+
+**Availability**: *when launched in a course that has a term*
```

Comment:
You should add an example term name below

Reply:
The modifications to this file are auto generated based on the comments in `lib/lti/variable_expander.rb`. You should be able to just run the api docs rake task to get the changes: `bundle exec rake doc:api`.

Reply:
fixed